tests/sentry/receivers/test_sentry_apps.py

```diff
@@ -46,13 +48,13 @@ def setUp(self):
 
         self.login_as(self.user)
 
-    def resolve_issue(self, _data=None):
+    def update_issue(self, _data=None):
         data = {'status': 'resolved'}
```

Comment:
seems a lil strange that the default is `resolved` but I don't necessarily think we need to change that right now since all but one of the tests are for resolved.